evcouplings/couplings/mapping.py

```diff
@@ -307,6 +307,41 @@ def to_model(self, x):
         return self.__map(x, self.target_to_model)
 
 
+def map_positions(dataframe, mapper, column):
```

Comment:
I made this its own function because it's getting called in complex/protocol/map_frequencies_file. Essentially we needed to access the functionality to remap single columns on its own.